packages/compiler/core/program.ts

```diff
@@ -105,7 +106,10 @@ export async function createProgram(
   // Load additional imports prior to compilation
   if (options.additionalImports) {
     const importScript = options.additionalImports.map((i) => `import "${i}";`).join("\n");
-    const sourceFile = createSourceFile(importScript, `__additional_imports`);
+    const sourceFile = createSourceFile(
```

Comment:
Switching to the system agnostic path utils found this issue where using `--import` would import relative to the current working directory and not where the main file is which I think is was a bug

Reply:
This is however causing some issues in the `cadl-azure` repo where we do `--import @azure-tools/cadl-autorest` for the core/samples where that package is of course not installed. Was working before as it was looking at the current working directory where it was available.